openff/bespokefit/workflows/bespoke.py

```diff
@@ -415,88 +398,88 @@ def _optimization_schema_from_records(
                 records_of_type
             )
 
-        smirks_gen = self._get_smirks_generator()
-        bespoke_parameters = []
-
-        for record, record_molecule in records:
-
-            if not isinstance(record, TorsionDriveRecord):
-                raise NotImplementedError("Only torsion drives are supported.")
-
-            dihedrals = record.keywords.dihedrals
-            assert len(dihedrals) == 1, "only 1D torsion drives are supported"
-
-            bespoke_parameters.extend(
-                smirks_gen.generate_smirks_from_molecule(
-                    molecule=record_molecule,
-                    central_bond=(dihedrals[0][1], dihedrals[0][2]),
-                )
-            )
+        # smirks_gen = self._get_smirks_generator()
+        # bespoke_parameters = []
+
+        # for record, record_molecule in records:
+        #
+        #     if not isinstance(record, TorsionDriveRecord):
+        #         raise NotImplementedError("Only torsion drives are supported.")
+        #
+        #     dihedrals = record.keywords.dihedrals
+        #     assert len(dihedrals) == 1, "only 1D torsion drives are supported"
+        #
+        #     bespoke_parameters.extend(
+        #         smirks_gen.generate_smirks_from_molecule(
+        #             molecule=record_molecule,
+        #             central_bond=(dihedrals[0][1], dihedrals[0][2]),
+        #         )
+        #     )
```

Comment:
Hmm I think in the world where we don't provide explicit parameters we'd need the fragmentation step to consider what local QC data has been provided and don't try and generate new fragments for the existing torsion drives?